packages/react-components/react-provider/src/components/FluentProvider/useFluentProviderContextValues.ts

```diff
@@ -26,6 +26,7 @@ export function useFluentProviderContextValues_unstable(state: FluentProviderSta
     overrides_unstable,
     provider,
     textDirection: dir,
+    iconDirection: {},
```

Comment:
This probably needs to be

```suggestion
    iconDirection: { textDirection: dir },
```

Reply:
```ts
const iconDirection = React.useMemo(() => ({ textDirection: dir }), [dir]);
```

Actually, there should be a `React.memo()` call as `IconDirectionContextValue` is an object.